runtime/experimental/versions.ini

```diff
@@ -0,0 +1,127 @@
+[libs]
+; https://github.com/madler/zlib/releases
+; Needed for:
+;   - openssl
+;   - curl
+;   - pcre
+zlib = "1.2.11"
+
+; https://github.com/openssl/openssl/releases
+; Needs:
+;   - zlib
+; Needed by:
+;   - curl
+;   - php
+openssl = "1.1.1a"
+
+; https://github.com/libssh2/libssh2/releases
+; Needs:
+;   - None
+; Needed by:
+;   - curl
+libssh2 = "1.8.0"
+
+; https://github.com/curl/curl/releases
+; Needs:
+;   - OpenSSL
+;   - zlib
+;   - libssh2
+; Needed by:
+;   - poppler
+;   - php
+curl = "7.63.0"
+
+; https://github.com/nih-at/libzip
+; Needs:
+;   - None
+; Needed by:
+;   - php
+libzip = "1.5.1"
+
+; http://www.linuxfromscratch.org/blfs/view/svn/general/nasm.html
+; Needs:
+;   - None
+; Needed by:
+;   - libjpeg-turbo
+nasm = "2.14"
+
+; https://github.com/GNOME/libxml2/releases
+; http://www.linuxfromscratch.org/blfs/view/svn/general/libxml2.html
+; Needs:
+;   - zlib
+; Needed by:
+;   - php
+libxml2 = "2.9.8"
+
+; https://github.com/libjpeg-turbo/libjpeg-turbo/releases
+; http://www.linuxfromscratch.org/blfs/view/svn/general/libjpeg.html
+; Needs:
+;   - nasm
+; Needed by:
+;   - webp
+libjpeg = "2.0.1"
+
+; https://libpng.sourceforge.io/
+; http://www.linuxfromscratch.org/blfs/view/svn/general/libpng.html
+; Needs:
+;   - None
+; Needed by:
+;   - php
+;   - webp
+libpng = "1.6.36"
+
+; https://storage.googleapis.com/downloads.webmproject.org/releases/webp/index.html
+; http://www.linuxfromscratch.org/blfs/view/8.0/general/libwebp.html
+; Needs:
+;   - libjpeg-turbo
+;   - png16
+; Needed by:
+;   - php
+libwebp = "1.0.1"
+
+; 
+; https://github.com/postgres/postgres/releases
+; Needs:
+;   - 
+; Needed by:
+;   - php
+postgres = "9.6.11"
+
+; https://github.com/jedisct1/libsodium/releases
+; Needs:
+;   - 
+; Needed by:
+;   - php
+sodium = "1.0.16"
+
+[php73]
+php = "7.3.0"
+imagemagick = "7.0.8-20"
+imagick = "3.4.3"
+memcached = "1.0.18"
+php_memcached = "3.1.3"
+
+[php72]
+php = "7.2.13"
+imagemagick = "7.0.8-20"
+imagick = "3.4.3"
+memcached = "1.0.18"
+php_memcached = "3.1.3"
+
+[php71]
+php = "7.1.25"
+imagemagick = "7.0.8-20"
+imagick = "3.4.3"
+memcached = "1.0.18"
+php_memcached = "3.1.3"
+
+[php70]
+php = "7.0.33"
+imagemagick = "7.0.8-20"
+imagick = "3.4.3"
+memcached = "1.0.18"
+php_memcached = "3.1.3"
```

Comment:
We can drop PHP 7.0 as it's not supported anymore.

Reply:
Done in commit https://github.com/mnapoli/bref/pull/132/commits/9a72b62ca1ed3d89d9e367f9982f5d919677574c